app/controllers/billing_expenses_controller.rb

```diff
@@ -11,7 +11,8 @@ def show
     respond_to do |format|
       format.html
       format.pdf do
-        render pdf: pdf_file_name, layout: 'pdf.pdf', encoding: 'UTF-8'
+        render pdf: pdf_file_name, layout: 'pdf.pdf',
+        template: 'billing_expenses/show.html.slim', encoding: 'UTF-8'
```

Comment:
And what is the advantage? And what was the bug?

Reply:
If you try to download a billing expense pdf on staging, you'll see that there is an error message. Locally it results in a missing partial error.